src/main/java/org/myrobotlab/document/transformer/XPathExtractor.java

```diff
@@ -132,7 +132,7 @@ private void processXml(String xml, Document doc) throws SAXException, IOExcepti
     try {
       fstream = new FileInputStream(filename);
     } catch (FileNotFoundException e) {
-      System.out.println("XPATH Extractor config file not found: " + filename);
+      log.warn("XPATH Extractor config file not found: {}" , filename);
       e.printStackTrace();
       return null;

```

Comment:
Stacktrace will go to the log if you 'do not' put a auto formatter bracket for it .. like this
log.warn("XPATH Extractor config file not found: {} " , filename, e);